src/Engine/Scene/Scene.cpp

```diff
@@ -40,6 +41,10 @@ Scene::Scene() :
     sysI = typeid(PathfindingSystem);
     m_compRefsBySysT[sysI].reset(new std::vector<Component *>());
     m_pathfindingSystem.reset(new PathfindingSystem(*m_compRefsBySysT[sysI]));
+
+	sysI = typeid(SoundSystem);
```

Comment:
Duplicated code?

Reply:
Yes. Honestly, I don't really get what the code does, but if every system is doing it, I figured SoundSystem should do so as well.